sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/TaskResourceMapper.java

```diff
@@ -17,7 +16,7 @@
  * Handles cross-referencing a preexisting {@link Protos.TaskInfo}'s current {@link Protos.Resource}s against a set
  * of expected {@link ResourceSpec}s for that task.
  */
-class TaskResourceMapper {
+public class TaskResourceMapper {
```

Comment:
Is this actually used outside of this package now? Didn't see any external usage in this change. But feel free to ignore if external usage is coming soon.